hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/management/GetCPMembersMessageTask.java

```diff
@@ -36,6 +37,8 @@
 
 public class GetCPMembersMessageTask extends AbstractAsyncMessageTask<Void, List<SimpleEntry<UUID, UUID>>> {
 
+    public static final Permission REQUIRED_PERMISSION = new ManagementPermission("cluster.getCPMembers");
```

Comment:
I'm not sure this one logically belongs to the `cluster` permission set since CP is a separate subsystem. Maybe it's better to consult with a CP expert.

Reply:
I agree with Zoltan. It might be better to group cp related things under one set since there are more than one operations that belong to the group.